dirsrvtests/tests/suites/paged_results/paged_results_test.py

```diff
@@ -510,11 +508,10 @@ def test_search_with_timelimit(topology_st, create_user):
     finally:
         del_users(users_list)
 
-#unstable or unstatus tests, skipped for now
-@pytest.mark.flaky(max_runs=2, min_passes=1)
+
 @pytest.mark.parametrize('aci_subject',
                          ('dns = "{}"'.format(HOSTNAME),
-                          'ip = "{}"'.format(IP_ADDRESS)),
+                          'ip = "{}" or ip = "::1"'.format(IP_ADDRESS)),
```

Comment:
This test is still failing for me. The test with IP is pass. The test with 'dns'  with localhost or with 'localhost or localhost.localdomain' is failing. When evaluating the ACI the server selects the aci but evaluation of the dns bind rule does not match. I do not know the reason. I guess it is reproducible and attaching to LASDnsEval would give us the culprit.
I agree with the changes in this PR, but on my laptop evaluation of dns constantly fails

Reply:
For methe  paged results (ip and dns) now passes on my laptop, but "dns" still fails in a container :-(   I'm working on it now (that's why there are all these commits to this PR)